charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -113,15 +113,23 @@ spec:
         command:
         - etcdbrctl
         - server
+{{- if .Values.etcd.defragmentationSchedule }}
         - --defragmentation-schedule={{ .Values.etcd.defragmentationSchedule }}
+{{- end }}
+{{- if .Values.backup.fullSnapshotSchedule }}
         - --schedule={{ .Values.backup.fullSnapshotSchedule }}
+{{- end }}
         - --garbage-collection-policy={{ .Values.backup.garbageCollectionPolicy }}
```

Comment:
Why not a check for this too?

Reply:
For `garbageCollectionPolicy`, we get a default value from values.yaml if its absent.